### PR TITLE
test: replace Popconfirm  part test with testing lib

### PR DESCRIPTION
**Pull request by lzm0x219**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
> Popconfirm › should show overlay when trigger is clicked

挂了……

**Comment by 0x219** [comment]
> > Popconfirm › should show overlay when trigger is clicked
> 
> 挂了……

是的 还需要改一下

File: components/popconfirm/__tests__/index.test.tsx [hunk 1]
```diff
@@ -89,31 +91,35 @@ describe('Popconfirm', () => {
...
     expect(ref.current.getPopupDomNode()).toBeTruthy();
     expect(ref.current.getPopupDomNode().className).not.toContain('ant-popover-hidden');
-    popconfirm.setProps({ visible: false });
-    popconfirm.update(); // https://github.com/enzymejs/enzyme/issues/2305
+    popconfirm.rerender(renderPopconfirm(false));
     act(() => {
       jest.runAllTimers();
     });
-    expect(popconfirm.find('Trigger').props().popupVisible).toBe(false);
+    expect(ref.current.props.visible).toBe(false);
```
Comment (afc163) [inline components/popconfirm/__tests__/index.test.tsx:114]: 检测 className 吧，之前检测内部 props 的其实也不准确。

Comment (0x219) [reply components/popconfirm/__tests__/index.test.tsx:114]: 好的

**Pull request closed without merging**